finance-reports/config.py

```diff
@@ -0,0 +1,76 @@
+EXPENSES_FIELD_MAPPING = {
+    "Fund@Code": "fund",
+    "Fund@Long Name": "fund_name",
+    "Department@Dept": "department",
+    "Department@Long Name": "department_name",
+    "Unit@Unit Code": "unit",
+    "Unit@Long Name": "unit_name",
+    "Object Code Category@Code": "object_category_code",
+    "Object Code Category@Long Name": "object_category_name",
+    "Object Code@Code": "object_code",
+    "Object Code@Long Name": "object_name",
+    "Object Code - Spending Plan@Code": "object_spend_plan_code",
+    "Object Code - Spending Plan@Long Name": "object_spend_plan_name",
+    "Budget YTD with Period Cutoff": "budget_ytd_with_period_cutoff",
+    "Proposed Budget CYE Amount FY & BFY Prompted (Phase 6)": "proposed_budget_cye_amount",
+    "Expenses MTD with Period Cutoff": "expenses_mtd",
+    "Expenses YTD with Period Cutoff": "expenses_ytd",
+    "Encumbrance YTD with Period Cutoff": "encumbrance_ytd",
+    "Unobligated": "unobligated",
+    "% Obligated": "percent_obligated",
+}
+
+REVENUE_FIELD_MAPPING = {
+    "Fund@Code": "fund",
+    "Fund@Long Name": "fund_name",
+    "Department@Dept": "department",
+    "Department@Long Name": "department_name",
+    "Unit@Unit Code": "unit",
+    "Unit@Long Name": "unit_name",
+    "Revenue Source Category@Code": "revenue_source_category_code",
+    "Revenue Source Category@Long Name": "revenue_source_category_name",
+    "Revenue Source@Code": "revenue_source_code",
+    "Revenue Source@Long Name": "revenue_source_name",
+    "Revenue Budget YTD with Period Cutoff": "revenue_budget_ytd",
+    "Proposed Revenue Amount CYE Prompted FY & BFY (Phase 6)": "revenue_cye",
+    "Revenue MTD with Period Cutoff": "revenue_mtd",
+    "Revenue YTD with Period Cutoff": "revenue_ytd",
+    "Revenue ITD with Period Cutoff": "revenue_itd",
+    "Unrecognized Amount": "unrecognized_amount",
+    "% Unrecognized": "percent_unrecognized",
+}
+
+EXPENSES_NUMERIC_COLS = [
+    "budget_ytd_with_period_cutoff",
+    "proposed_budget_cye_amount",
+    "expenses_mtd",
+    "expenses_ytd",
+    "encumbrance_ytd",
+    "unobligated",
+    "percent_obligated",
+]
+
+REVENUE_NUMERIC_COLS = [
+    "revenue_ytd",
+    "revenue_budget_ytd",
+    "revenue_cye",
+    "revenue_mtd",
+    "revenue_itd",
+    "unrecognized_amount",
+    "percent_unrecognized",
+]
+
+MONTH_LOOKUP = {
+    1: "Jan",
+    2: "Feb",
+    3: "Mar",
+    4: "Apr",
+    5: "May",
+    6: "Jun",
+    7: "Jul",
+    8: "Aug",
+    9: "Sep",
+    10: "Oct",
+    11: "Nov",
+    12: "Dec",
+}
```

Comment:
This is totally fine and doesn't need changing at all. ✅

While I was waiting for this to run during my test, I poked around seeing if I could abstract this object back into the realm of a library. `datetime` and its `strftime` method can get at these month abbreviations if you want. 

```
from datetime import datetime

# To get month name by its number
month_num = 4  # April
month_name = datetime(2022, month_num, 1).strftime('%b')
print("April, for example: ", month_name)  # Output: 'Apr'

months = {
    month: datetime(2022, month, 1).strftime('%b')
    for month in range(1, 13)
}

print()

print("The whole object: ", months)
```

Reply:
ty Frank, very helpful! Will drop that lookup table in favor of this